CalibCalorimetry/HcalAlgos/plugins/HcalPulseContainmentTest.cc

```diff
@@ -65,6 +65,9 @@ void HcalPulseContainmentTest::analyze(const edm::Event& iEvent, const edm::Even
   double corr4 = manager->correction(he1, 4, fixedphase_ns, fc);
   assert(corr4 == corr1);
   std::cout << corr1 << " " << corr2 << " " << corr3 << " " << corr4 << " " << std::endl;
+  // test 1TS correction at high energy
+  double corr5 = manager->correction(hb1, 1, fixedphase_ns, 100000.);
+  std::cout << corr5 << std::endl;
```

Comment:
My understanding is that the use of std::cout is discouraged. Please use the relevant MessageLogger instead, https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideMessageLogger